lib/new_relic/agent/instrumentation/ruby_openai/instrumentation.rb

```diff
@@ -30,6 +30,7 @@ def embeddings_instrumentation(parameters)
       segment = NewRelic::Agent::Tracer.start_segment(name: EMBEDDINGS_SEGMENT_NAME)
       record_openai_metric
       event = create_embeddings_event(parameters)
+      event = remove_input(event) if !record_content_enabled?
```

Comment:
Does either the spec or the code encourage this approach of conditionally removing content? If not, I wonder if conditionally adding that content beforehand would be better.

Reply:
Thanks for your help @fallwith! We are now adding content vs removing.